console/db/migrations/20241205103951_2.1.0.sql

```diff
@@ -20,6 +20,35 @@ WHERE extension_name IN (
     'timescaledb'
 );
 
+
+-- Extends 20240520144338_2.0.0_initial_scheme_setup.sql#L217 with more cloud instance types
+-- Heztner price is for the region 'Geremany / Finland', other regions may vary in price.
+INSERT INTO public.cloud_instances (cloud_provider, instance_group, instance_name, cpu, ram, price_hourly, price_monthly, currency, updated_at) VALUES
+    ('hetzner', 'Small Size', 'CX22', 2, 4, 0.0074 , 4.59, '$', '2024-12-10'),    
+    ('hetzner', 'Small Size', 'CX32', 4, 8, 0.0127 , 7.59, '$', '2024-12-10'),
+    ('hetzner', 'Medium Size', 'CX42', 8, 16, 0.0304 , 18.59, '$', '2024-12-10'),
+    ('hetzner', 'Medium Size', 'CX52', 16, 32, 0.0611 , 36.09, '$', '2024-12-10'),
+    ('hetzner', 'Small Size', 'CPX31', 4, 8, 0.025 , 15.59, '$', '2024-12-10'),
+    ('hetzner', 'Medium Size', 'CPX41', 8, 16, 0.0464 , 28.09, '$', '2024-12-10'),
+    ('hetzner', 'Medium Size', 'CPX51', 16, 32, 0.0979 , 61.09, '$', '2024-12-10');
```

Comment:
A suggestion for indicating these are shared vCPU.

```suggestion
    ('hetzner', 'Small Size', 'CX22 (Shared vCPU)', 2, 4, 0.0074 , 4.59, '$', '2024-12-10'),    
    ('hetzner', 'Small Size', 'CX32 (Shared vCPU)', 4, 8, 0.0127 , 7.59, '$', '2024-12-10'),
    ('hetzner', 'Medium Size', 'CX42 (Shared vCPU)', 8, 16, 0.0304 , 18.59, '$', '2024-12-10'),
    ('hetzner', 'Medium Size', 'CX52 (Shared vCPU)', 16, 32, 0.0611 , 36.09, '$', '2024-12-10'),
    ('hetzner', 'Small Size', 'CPX31 (Shared vCPU)', 4, 8, 0.025 , 15.59, '$', '2024-12-10'),
    ('hetzner', 'Medium Size', 'CPX41 (Shared vCPU)', 8, 16, 0.0464 , 28.09, '$', '2024-12-10'),
    ('hetzner', 'Medium Size', 'CPX51 (Shared vCPU)', 16, 32, 0.0979 , 61.09, '$', '2024-12-10');
```

Reply:
This approach will cause errors during deployment because the server type will not be transmitted correctly. It’s better to add a shared_cpu column (boolean), which allows us to expand the API and properly display “Shared vCPU” in the UI.

Reply:
In that case disregard my suggestions 👍